generated/nifake/nifake/unit_tests/test_session.py

```diff
@@ -1405,6 +1405,49 @@ def test_nitclk_integration(self):
         with nifake.Session('dev1') as session:
             assert str(type(session.tclk)) == "<class 'nitclk.session.SessionReference'>"
 
+    def test_accept_time_values(self):
+        self.patched_library.niFake_AcceptListOfTimeValues.side_effect = self.side_effects_helper.niFake_AcceptListOfTimeValues
+        # Pass one argument as list and one as array.array
+        timestamps = [-1.5, 2.0]
```

Comment:
these aren't time values, they are floats.
Is this testing the conversion?

Reply:
By "time values" I mean number types representing time, as opposed to `timedelta` instance representing time.
I can rename the test method as `test_accept_time_as_floats`. 

This is testing the case where the converter does not have to convert anything, but just pass through the list of number types it gets.

Reply:
Updated the test names